tests/zeroconfig/windows/testdata/Dockerfile.pipeline.collector

```
@@ -0,0 +1,27 @@
+# escape=`
+
+ARG windowscontainer_version=ltsc2022
+FROM mcr.microsoft.com/windows/servercore:${windowscontainer_version}
+
+# On ltsc2019 RUN defaults to "powershell -Command {0}", while ltsc2022 RUN defaults to "cmd /S /C"
+# set the SHELL so it is consistent between both versions.
+SHELL [ "powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';" ]
+
+COPY docker-setup/ /setup
+COPY ./pipeline-collector.yaml /setup
+
+# WARNING: Passing access token as a build argument is to facilitate manual validation of this Dockerfile. It shouldn't be used in production.
+ARG verify_access_token=false
+ARG access_token=some_token
+ENV VERIFY_ACCESS_TOKEN=$verify_access_token
+ENV ACCESS_TOKEN_TMP=$access_token
+RUN `
+    $token = $Env:ACCESS_TOKEN_TMP; `
+    $collector_msi_path = (dir "c:\setup\splunk-otel-collector-*.msi").FullName; `
+    c:\setup\install.ps1 -access_token $token -msi_path $collector_msi_path -with_fluentd $false -ingest_url "http://10.1.1.1:4318"
```

mind configuring it to use a reserved/unused address in the off chance something leads it to report to the otlp sink directly?

```suggestion
    c:\setup\install.ps1 -access_token $token -msi_path $collector_msi_path -with_fluentd $false -ingest_url "http://192.0.2.1:12345"
```